backend/scripts/sync_prod_to_sqlite.py

```
@@ -0,0 +1,284 @@
+    sys.exit('Postgres container did not become ready.')
+
+
+def restore(dump_path):
+    log(f'Restoring {dump_path.name} into local Postgres')
+    run(['docker', 'cp', str(dump_path), f'{CONTAINER}:/tmp/dump.sql'])
+    # The dump carries --clean --if-exists, so restoring over an existing copy
+    # is fine; ON_ERROR_STOP=0 tolerates the drop statements on a fresh volume.
+    run([
+        'docker', 'exec', '-e', f'PGPASSWORD={PG_PASSWORD}', CONTAINER,
+        'psql', '-q', '-U', 'postgres', '-d', 'postgres',
+        '-v', 'ON_ERROR_STOP=0', '-f', '/tmp/dump.sql',
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/dump.sql"

(S108)